sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseNewActivity.java

```diff
@@ -80,8 +81,16 @@ public static void startActivityWithEmptyReportDate(Context fromActivity) {
 		BaseEditActivity.startActivity(fromActivity, CaseNewActivity.class, buildBundleWithEmptyReportDate());
 	}
 
-	public static void startActivityFromContact(Context fromActivity, String contactUuid) {
-		BaseEditActivity.startActivity(fromActivity, CaseNewActivity.class, buildBundleWithContact(contactUuid));
+	public static void startActivityFromContact(Context fromActivity, Contact contact) {
+		if (contact.getContactClassification() == ContactClassification.UNCONFIRMED) {
```

Comment:
```suggestion
		if (contact.getContactClassification() != ContactClassification.CONFIRMED) {
```

The other classifications also don't allow users to convert the contact

Reply:
Yes, but the NO_CONTACT classification can not happen at all because the button will not be displayed on this case.
That's why I thought that it's enough to check the UNCONFIRMED , but I can change as you've suggested. :)